src/deepsparse/loggers/async_logger.py

```diff
@@ -59,6 +60,12 @@ def log(self, identifier: str, value: Any, category: MetricCategories):
         )
         job_future.add_done_callback(_log_async_job_exception)
 
+    def __str__(self, level=0):
+        wrapper = textwrap.TextWrapper(initial_indent="\n" + "  " * level)
+        text = wrapper.fill(f"{self.__class__.__name__}:")
+        text += self.logger.__str__(level=level + 1)
+        return text
```

Comment:
thoughts on this instead?
```suggestion
    def __str__(self):
        child_str = textwrap.indent(str(self.logger), prefix="  ")
        return f"{self.__class__.__name__}:\n{child_str}"
```

Reply:
since we call `indent` over the `str` of the entire child, level propagation happens automatically, as if the current "level" is > 1, then this entire constructed string will be indented by its parent(s)